src/views/AIView.tsx

```diff
@@ -10,17 +10,19 @@ export function AIView({ selection }: { selection: string | null }) {
   }, [selection]);
 
   return (
-    <div>
+    <div className='flex flex-col h-full w-full'>
       <h1>AI Interactions</h1>
-      <p className="my-4 italic">Select some text in the editor to see it here.</p>
+      <div className='flex-1 overflow-scroll'>
```

Comment:
Do you think worth creating a wrapper component for a panel view with title so that we can reuse?

Reply:
Yes, I think it makes sense! I will update the PR

Reply:
As discussed offline, only AI Interactions and References could currently use a common wrapper, so creating the wrapper isn't that worth. We should open an issue to refactor the folders view to something more similar to VSCode, with the possibility to drag-and-drop in a specific folder, etc. The wrapper can then be implemented in the PR refactoring the view.